main.go

```diff
@@ -64,6 +64,9 @@ func main() {
 		if errors.Is(err, exit.ErrAborted) {
 			os.Exit(exit.StatusAborted)
 		}
+		if errors.Is(err, exit.ErrEmptyList) {
+			os.Exit(exit.StatusEmptyList)
+		}
```

Comment:
I would probably delete this and `exit.StatusEmptyList` and let it fallthrough to the default status of 1.

Reply:
The fallthrough would print the error.

Reply:
Ah totally I forgot! Apologies